apps/common/js/components/leap-hands-view.jsx

```diff
@@ -11,9 +11,10 @@ const SKIN_COLOR = 0x93603F;
 export default class LeapHandsView extends React.Component {
   componentDidMount() {
     const { handsOpacity } = this.props;
-    const renderer = new THREE.WebGLRenderer({alpha: true});
+    const renderer = new THREE.WebGLRenderer({ alpha: true });
     renderer.setClearColor(0x000000, 0);
     renderer.setSize(this.width, this.height);
+    this.setState({ renderer: renderer });
```

Comment:
When resizing the renderer I was attempting to resize the existing renderer that had been created previously. I'm not sure it is needed - will check into this.